torch/nn/utils/spectral_norm.py

```diff
@@ -34,40 +40,41 @@ def compute_weight(self, module):
         return weight, u
 
     def remove(self, module):
-        weight = module._parameters[self.name + '_orig']
+        weight = getattr(module, self.name)
         delattr(module, self.name)
         delattr(module, self.name + '_u')
         delattr(module, self.name + '_orig')
-        module.register_parameter(self.name, weight)
+        module.register_parameter(self.name, torch.nn.Parameter(weight))
```

Comment:
Isn't `weight` Parameter? (plz ignore this freely)

Reply:
While you have spectral norm (which you still do at this point), it is a Tensor (because it is computed / a buffer).